src/settings/apiSettings.ts

```diff
@@ -19,6 +19,7 @@ export const COLLECTION_WORLDS = "worlds";
 export const COLLECTION_WORLD_EVENTS = "worldEvents";
 export const COLLECTION_SEATED_USERS = "seatedUsers";
 export const COLLECTION_SEATED_USERS_CHECKINS = "recentSeatedUsers";
+export const COLLECTION_RETUNABLE_MEDIA_ELEMENTS = "retunableMediaElements";
```

Comment:
[NOTE]
We should've kept this list alphabetically sorted for easier visual search. I have failed to add a comment at the top like in other places.